stdpopsim/slim_engine.py

```diff
@@ -167,7 +208,8 @@
         for (i in 1:(num_epochs-1)) {
             g = G[i-1];
             for (j in 0:(num_populations-1)) {
-                if (N[i,j] != N[i-1,j]) {
+                if (N[i,j] != N[i-1,j] & N[i-1,j] != 0) {
```

Comment:
I don't have the big picture here, but is this right? Can't a population goes from size 0 to something positive?

Reply:
It turns out that setting population sizes in this case is redundant (they're all taken care of by either `sim.addSubpop()` or `sim.addSubpopSplit()`). I only just noticed though, due to explicitly counting the warning in the introduced tests.

Reply:
Maybe this warrants an explanatory comment.